source/templates/handlebars-basics.md

```diff
@@ -11,9 +11,19 @@ write any additional code to make sure it keeps up-to-date.
 
 ### Defining Templates
 
-By default, adjust your [application template](../the-application-template), that is created automatically for you and will be displayed on the page when your app loads.
+The first thing you should change is your [application template](../the-application-template) that is created
```

Comment:
One quibble here: most people getting started in an Ember app _won't_ modify their application template, since the default `{{outlet}}` is good enough. I'd argue that people should be starting with the `index` template.